### PR TITLE
GitHub biography integration

### PR DESCRIPTION
**Pull request by aeltyif**
I have added:
- Biography to be retrieved from the API as it was a static code
- Discount on each book from the API

I have modified:
- Coding styles AKA code clean up

Store Front: https://drive.google.com/open?id=1Cj03puCctoT-K9jkouH5zGfq5tC3TJdK
Clicking on any book: https://drive.google.com/open?id=1qmDondcyP5T4jWs87rTmdYGE5ALtTomK
Showing all books: https://drive.google.com/open?id=1rQfCrBoJlV5TSWrxzFE922I-F6zUDRIU
Author biography: https://drive.google.com/open?id=1W8-7gdmpVZZsT3ms17K3-gdFTOoZSozr

PS: I have tried to upload the images directly here but "something really bad happened" is what github is saying

### CODE REVIEW

**Comment by vladucu** [comment]
Hi Ahmad, 

This looks pretty sweet overall! 

I would love to see some screen-shots/recordings attached to this PR to showcase what's done here... it goes a long way for frontend PR reviews.

**Comment by aeltyif** [comment]
@vladucu Thanks Vlad, do you think a screen casting with some audio will do the trick ?
Also we are show casing the frontend not the code itself but what it represents correct ?

**Comment by vladucu** [comment]
> @vladucu Thanks Vlad, do you think a screen casting with some audio will do the trick ?
Also we are show casing the frontend not the code itself but what it represents correct ?

No strong preference, up to you really.
Yeah, to showcase the frontend not the code itself....thanks

**Review by vladucu — APPROVED** [review]
This looks pretty good to me Ahmad! Nicely done

Let's merge this in `master` and consider this part complete
🎉 👏

File: app/templates/author.hbs [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 
-<strong>Biography</strong>: Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+<strong>Biography</strong>: {{model.biography}}
 
 <h4>Published titles:</h4>
 <ul>
-{{#each model.books as |book|}}
-  {{book-cover book=book blurBackground=(route-action "blurBackground")}}
-{{/each}}
+  {{#each model.books as |book|}}
```
[the remark this answers is not in the document]
Comment (aeltyif) [reply app/templates/author.hbs]: Thank you 👍

**Pull request merged**